CHANGELOG.md

```diff
@@ -14,6 +14,10 @@ Types of changes
 - `Fixed` for any bug fixes.
 - `Security` in case of vulnerabilities.
 
+## [2.6.2]
+
+- `Changed` Add columns info and export type with command `lino table extract`, columns and keys are in order of database.
```

Comment:
This should be documented as a new feature with `Added` instead of `Changed`
The `--only-table` is there to keep the previous behavior